packages/reporter/src/main.tsx

```diff
@@ -30,19 +30,19 @@ interface BaseReporterProps {
   events: Events
   error?: RunnablesErrorModel
   resetStatsOnSpecChange?: boolean
-  renderReporterHeader?: (props: ReporterHeaderProps) => JSX.Element;
+  renderReporterHeader?: (props: ReporterHeaderProps) => JSX.Element
   spec: Cypress.Cypress['spec']
   experimentalStudioEnabled: boolean
   /** Used for component testing front-end */
   specRunId?: string | null
 }
 
 export interface SingleReporterProps extends BaseReporterProps{
```

Comment:
I wonder if there is a way to lint for 

`BaseReporterProps {` instead of `BaseReporterProps{`

Reply:
I bet the new rules I've built for our greenfield projects does lint for that. That isn't in this PR though.